cluster-provision/k8s/scripts/node01.sh

```diff
@@ -10,14 +10,20 @@ done
 
 kubeadm init --config /etc/kubernetes/kubeadm.conf
 
+default_cidr="192.168.0.0/16"
+pod_cidr="10.244.0.0/16"
 version=`kubectl version --short --client | cut -d":" -f2 |sed  's/ //g' | cut -c2- | cut -d"." -f2`
-flannel_manifest="/tmp/flannel.yaml"
-if [[ $version -ge "16" ]]; then
-    flannel_manifest="/tmp/flannel-ge-16.yaml"
+
+network_plugin_manifest="/tmp/flannel.yaml"
```

Comment:
I have do a search and no one is using node01.sh, I think it's safe to delete so we don't have the flannel/calico installation logic duplicated.

Reply:
I think this script is being used [here](https://github.com/kubevirt/kubevirtci/blob/db3913a3a919bbc20c3b7044dffc70c009fdc3bd/cluster-provision/cli/cli#L242)

Reply:
Then let's put the plugin selection at common place, the logic is big enough now for that since we have the default cidr replacement and all.

Reply:
Done